.github/workflows/stack-init.yml

```diff
@@ -0,0 +1,85 @@
+name: "Setup and deploy farm stack on azure"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  stack-initialization:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: use node.js v14
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - name: use python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - name: Stacks framework code generation
+        run: |
+          pip install cookiecutter
+          cookiecutter --no-input gh:prabum3/farm-stack-template project_slug=${{ secrets.WEB_APP_NAME }} container_registry=${{ secrets.AZURE_CONTAINER_REGISTRY }}
+
+      - name: Reorganize files
+        run: |
+          mv -v ./${{ secrets.WEB_APP_NAME }}/*  .
+          rm -rf ./${{ secrets.WEB_APP_NAME }}
+
+      - name: Configure git
+        run: |
+          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
+          git config --global user.name "${{ github.actor }}"
+          git remote set-url origin https://${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}.git
+
+      - name: Commit generated code and initialize workflows
+        run: |
+          git add .
+          git commit -m "Generate framework code and initialize workflows"
+          git push
+
+      - name: Azure login
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Login into container registry
+        run: |
+          az acr login --name ${{ secrets.AZURE_CONTAINER_REGISTRY }}
+
+      - name: Build React App
+        run: |
+          cd ./client
+          npm install
+          npm run build
+
+      - name: Build and push docker image (nginx-reverse-proxy)
+        run: |
+          cd ./nginx/production
+          IMAGE_NAME=${{ secrets.AZURE_CONTAINER_REGISTRY }}/${{ secrets.WEB_APP_NAME }}-nginx-react-image:latest
+          echo "building image $IMAGE_NAME"
+          docker image build -t $IMAGE_NAME .
+          docker push $IMAGE_NAME
+
+      - name: Build and push docker image (fastapi)
+        run: |
+          cd ./server
+          IMAGE_NAME=${{ secrets.AZURE_CONTAINER_REGISTRY }}/${{ secrets.WEB_APP_NAME }}-fastapi-image:latest
+          echo "building image $IMAGE_NAME"
+          docker image build -t $IMAGE_NAME .
+          docker push $IMAGE_NAME
```

Comment:
Curious if you could modify the workflows in the PR to take advantage of reusable workflows:
https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#calling-a-reusable-workflow

Reply:
Merging this PR now, will iterate on having reusable workflows